src/components/Accordion/index.tsx

```diff
@@ -28,43 +29,94 @@ const CustomAccordion: React.FC<AccordionData> = ({
   details,
   index,
 }) => {
+  const { colorMode } = useColorMode();
+  const isDarkTheme = colorMode === 'dark';
+  
+  // Create a theme that adapts to dark/light mode
+  const theme = createTheme({
+    components: {
+      MuiAccordion: {
+        styleOverrides: {
+          root: {
+            backgroundColor: isDarkTheme ? '#1b1b1d' : 'var(--ifm-background-color)',
+            color: isDarkTheme ? 'var(--ifm-color-primary-lightest)' : '#1b1b1d',
```

Comment:
Maybe it's cleaner to use a `var(...)` here as well like you do below? But not a big deal

Reply:
Or maybe save the hex color code in a variable:

```js
const backgroundColorDark = '#1b1b1d';
```